tests/default_vars.sh

```diff
@@ -449,3 +453,106 @@ export_35d_run ()
 export CNTL_DIR=""
 export LIST_FILES=""
 }
+export_datm ()
+{
+export FV3=false
+export S2S=false
+export DATM=true
+export DAYS=1
+export FHMAX=24
+export WLCLK=30
+export THRD=1
+export FHROT='0'
+
+# atm/ocn/ice resolution
+# GEFS
+export DATM_SRC="GEFS"
+export FILENAME_BASE='gefs.'
+export IATM=1536
+export JATM=768
+export ATMRES='C96'
+export OCNRES='100'
+export ICERES='1.00'
+export NX_GLB=360
+export NY_GLB=320
+
+# nems.configure
+export MED_restart_data=''
```

Comment:
If this is not used, it should be removed.

Reply:
It is removed.

Reply:
Have you pushed these changes? I'm looking at your commit ffbf14d and I don't see any changes made yet.